bedrock/firefox/templates/firefox/share.html

```diff
@@ -0,0 +1,88 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import card, callout, split with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ main_title }}{% endblock %}
+{% block page_desc %}{{ page_description }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/share-page/fx-share-the-love.png') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_share') }}
+{% endblock %}
+
+{% if LANG == "de" %}
+  {% set main_title = "Hol deine Freund*innen ins <br><strong>Team Firefox</strong>"|safe %}
+  {% set page_description = "Wir glauben, als gute*r Freund*in solltest du sie auf die sonnige Seite des Internets einladen. Wir haben da mal was vorbereitet." %}
+  {% set main_tagline = "Wir haben eine kleine Nachricht vorbereitet, die du mit deinen Liebsten teilen kannst, damit du dir nicht selbst etwas ausdenken musst. Wird keiner merken – versprochen!" %}
+  {% set share_message = "Hey! Ich wollte dir nur sagen, dass du echt schöne Augen hast. Ihre perfekte Form erinnert mich an das Firefox-Logo. Kennst du Firefox? Das ist dieser private, super verlässliche Browser. Eigentlich echt gut, hier ist ein Link, falls du dir den mal angucken willst: <a href='https://mzl.la/ctd-de'>mzl.la/ctd-de</a>. <br><br> Wie gehts dir eigentlich?"|safe%}
```

Comment:
**suggestion:** I asked about this in the slack convo, but I find the share message a little odd here in today's social media dumpster fire. I'll defer to others on what to do here if anything.

Reply:
Haha EU team is well-aware and they did say that it sounds less-creepy in French and German languages and then we had a whole discussion on how English is a prudish language 😆